recipes/podofo/all/conanfile.py

```diff
@@ -35,26 +40,18 @@ class PodofoConan(ConanFile):
         "with_tiff": True,
         "with_png": True,
         "with_unistring": True,
+        "with_tools": False,
     }
 
-    generators = "cmake", "cmake_find_package"
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
-
-    @property
-    def _build_subfolder(self):
-        return "build_subfolder"
+    generators = "CMakeToolchain", "CMakeDeps", "VirtualRunEnv"
```

Comment:
You cant use both the attribute and the method at the same time. I think it's getting confused

```suggestion
    generators = "VirtualRunEnv"
```

Reply:
It is ok to define both the default generator CMakeToolchain and use the CMakeToolchain tool module .
If you look at all the other recipes in conan center (still using the conan v1 cmake tool), they both define the cmake generator and use the cmake tool module.
I followed the example here: https://github.com/conan-io/conan-center-index/pull/12617/ and trying to stay consistent with the rest of the recipes.

Reply:
Ok I removed them